dev_environment/cfg/payload.json

```diff
@@ -1,4 +1,77 @@
-}
+[
+  {
+    "host_name" : "l3af-local-test",
+    "Iface" : "enp0s3",
+    "bpf_programs" : {
+      "xdp_ingress" : [
+        {
+          "name": "ratelimiting",
+          "seq_id": 1,
+          "artifact": "l3af_ratelimiting.tar.gz",
+          "map_name": "/sys/fs/bpf/xdp_rl_ingress_next_prog",
+          "cmd_start": "ratelimiting",
+          "version": "latest",
+          "is_user_program": true,
+          "admin_status": "enabled",
+          "ebpf_type": "xdp",
+          "cfg_version": 1,
+          "start_args": { "ports": "8080,8081", "rate": "2" },
+          "monitor_maps": [
+            { "name": "rl_drop_count_map", "key": 0, "aggregator": "scalar"},
+            { "name": "rl_recv_count_map", "key": 0, "aggregator": "max-rate"}
+          ]
+        },
+        {
+          "name": "connection-limit",
+          "seq_id": 2,
+          "artifact": "l3af_connection_limit.tar.gz",
+          "map_name": "/sys/fs/bpf/xdp_cl_ingress_next_prog",
```

Comment:
The map name looks like the full path, not just the map name itself.
Could we just make it the map name itself (i.e., "xdp_cl_ingress_next_prog" where path "/sys/fs/bpf/" is implied on Linux), since the path will be different between Linux vs Windows.  That is, allow relative paths not just absolute paths.

Reply:
Yes, we can do that. Here TC  programs map file path ```/sys/fs/bpf/tc/globals/``` and XDP programs map file path  ```/sys/fs/bpf/```.

Reply:
We have opened issue in l3afd #33